express/parsers/mixins/ionic.py

```diff
@@ -298,6 +298,22 @@ def space_group_symbol(self):
         """
         pass
 
+    @abstractmethod
+    def point_group_symbol(self):
```

Comment:
We need to be consistent with naming and use `symmetry_symbol_point_group` and space_group accordingly